nifi-commons/nifi-record-path/pom.xml

```diff
@@ -79,7 +79,7 @@
         <dependency>
             <groupId>com.github.ben-manes.caffeine</groupId>
             <artifactId>caffeine</artifactId>
-            <version>2.6.2</version>
+            <version>2.8.1</version>
```

Comment:
I've run into issues with caffeine API compatibility a while ago ( long enough that my prior experience may be totally invalid today ). Do you feel confident these won't cause any issues? I have some flows where I can try these changes out with my record paths ( most seem like they are already covered by unit tests ) -- so if you can resolve conflicts i'd be happy to take some cycles to try this out.

Reply:
no i have no confidence beyond what our tests offer